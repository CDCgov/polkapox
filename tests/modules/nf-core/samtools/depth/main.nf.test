nextflow_process {

    name "Test Process SAMTOOLS_DEPTH"
    script "modules/nf-core/samtools/depth/main.nf"
    process "SAMTOOLS_DEPTH"

    test("Should run without failures") {

        when {
            params {

            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = [ [ meta: 'test_sub'], file("${projectDir}/tests/nf-test-data/test_sub.bam") ]
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
