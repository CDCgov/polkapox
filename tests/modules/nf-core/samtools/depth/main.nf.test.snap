{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "tsv": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-06-27T13:37:39.775149552"
    }
}